doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -301,7 +301,9 @@ Debug libraries
 Modem libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`pdn_readme` library:
+
+  * Updated the library to allow PDP_TYPE only configuration in the :c:func:`pdn_ctx_configure` function.
```

Comment:
What is PDP_TYPE? It seems like a variable, but a quick search doesn't turn up anything?

Reply:
Please refer to [link](https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/packet_domain/cgdcont_set.html?cp=2_1_6_0_0)